modin/pandas/test/test_io.py

```diff
@@ -382,6 +391,46 @@ def test_read_csv_gzip():
     df_equals(modin_df, pandas_df)
 
 
+def test_read_csv_bz2(make_csv_file):
+    make_csv_file(compression="bz2")
+    bz2_path = TEST_CSV_FILENAME + ".bz2"
+
+    pandas_df = pandas.read_csv(bz2_path)
+    modin_df = pd.read_csv(bz2_path)
+    df_equals(modin_df, pandas_df)
+
+    pandas_df = pandas.read_csv(bz2_path, compression="bz2")
+    modin_df = pd.read_csv(bz2_path, compression="bz2")
+    df_equals(modin_df, pandas_df)
+
+
+def test_read_csv_xz(make_csv_file):
+    if not PY2:
+        make_csv_file(compression="xz")
+        xz_path = TEST_CSV_FILENAME + ".xz"
+
+        pandas_df = pandas.read_csv(xz_path)
+        modin_df = pd.read_csv(xz_path)
+        df_equals(modin_df, pandas_df)
+
+        pandas_df = pandas.read_csv(xz_path, compression="xz")
+        modin_df = pd.read_csv(xz_path, compression="xz")
+        df_equals(modin_df, pandas_df)
+
+
+def test_read_csv_zip(make_csv_file):
```

Comment:
Perhaps we can add a few more cases here so that we can hit these lines in the diff that are getting missed?

https://codecov.io/gh/modin-project/modin/pull/722/diff#D1-70...83

https://github.com/modin-project/modin/blob/12a895195a51b7846fe16e46195b9ee6beede2ec/modin/engines/ray/generic/io.py#L70-L86

Reply:
The ZipExtFile only supports .seek() starting python3.7 which is not a Travis environment so it keeps defaulting to the pandas implementation when running the unit test .

Reply:
ah ok that makes sense. Thanks!